### PR TITLE
Porting Python2 to Python3 and Fix name variable

### PR DESCRIPTION
**Pull request by drego85**

### CODE REVIEW

File: CONTRIB/PRODUCER/scripts/CS_build_stix-from_files.py [hunk 1]
```diff
@@ -29,39 +30,40 @@
...
         return []
 
-def main(argv):
 
+def main(argv):
     ######################################################################
     # Se non impostati da command line vengono utilizzati i seguenti valori per TITLE, DESCRIPTION, IDENTITY
     # Il title e' ID univoco della minaccia (es. Cobalt / Danabot / APT28)
-    TITLE = "Test"
+    TITLE = "LokiBot Italy - Payment Copy"
```
Comment (CyberSaiyanIT) [inline CONTRIB/PRODUCER/scripts/CS_build_stix-from_files.py]: can you set back to "Test" pls

File: CONTRIB/PRODUCER/scripts/CS_build_stix-from_files.py [hunk 2]
```diff
@@ -29,39 +30,40 @@
...
 
     # La description strutturiamola come segue
     # <IOC PRODUCER> - <Descrizione della minaccia/campagna> - <URL (if any)>
-    DESCRIPTION = "Cyber Saiyan - Test - https://infosharing.cybersaiyan.it"
+    DESCRIPTION = "Test Sharing"
 
     # La sorgente che ha generato l'IoC con riferimento a Cyber Saiyan Community 
-    IDENTITY = "Cyber Saiyan Community"
+    IDENTITY = "D3Lab"
```
Comment (CyberSaiyanIT) [inline CONTRIB/PRODUCER/scripts/CS_build_stix-from_files.py]: can you set back to "Cyber Saiyan Community" pls?

File: CONTRIB/PRODUCER/scripts/CS_build_stix-from_files.py [hunk 3]
```diff
@@ -29,39 +30,40 @@
...
     ######################################################################
     # Se non impostati da command line vengono utilizzati i seguenti valori per TITLE, DESCRIPTION, IDENTITY
     # Il title e' ID univoco della minaccia (es. Cobalt / Danabot / APT28)
-    TITLE = "Test"
+    TITLE = "LokiBot Italy - Payment Copy"
 
     # La description strutturiamola come segue
     # <IOC PRODUCER> - <Descrizione della minaccia/campagna> - <URL (if any)>
-    DESCRIPTION = "Cyber Saiyan - Test - https://infosharing.cybersaiyan.it"
+    DESCRIPTION = "Test Sharing"
```
Comment (CyberSaiyanIT) [inline CONTRIB/PRODUCER/scripts/CS_build_stix-from_files.py]: can you set back to "Cyber Saiyan - Test - https://infosharing.cybersaiyan.it"?

**Comment by drego85** [comment]
Now I close, a more complete PR will arrive.
Sorry for error values TITLE, IDENTITY and DESCRIPTION.

**Pull request closed without merging**